docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "2.1"
+services:
+  studio-frontend:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    command: bash -c "cd /edx/src/studio-frontend && npm run start"
+    #depends_on:
+    #  - studio
+    volumes:
+      - .:/edx/src/studio-frontend
```

Comment:
To start: `git clean -xfd; docker-compose -f docker-compose.yml up -d studio-frontend`

The `git clean` helps in case the installed `node_modules` gets back into your host machine after a build, that was causing me so much trouble.